seqr/management/commands/transfer_families_to_different_project.py

```diff
@@ -1,21 +1,39 @@
+from collections import defaultdict
+
 from django.core.management.base import BaseCommand
 
 from seqr.models import Project, Family, VariantTag, VariantTagType, Sample
 from seqr.utils.search.utils import backend_specific_call
+from seqr.views.utils.airflow_utils import trigger_airflow_delete_families, DagRunningException
 
 import logging
 logger = logging.getLogger(__name__)
 
 
-def _disable_search(families):
+def _disable_search(families, from_project):
     search_samples = Sample.objects.filter(is_active=True, individual__family__in=families)
     if search_samples:
         updated_families = search_samples.values_list("individual__family__family_id", flat=True).distinct()
+        updated_family_dataset_types = list(search_samples.values_list('dataset_type', 'individual__family__guid').distinct())
         family_summary = ", ".join(sorted(updated_families))
         num_updated = search_samples.update(is_active=False)
         logger.info(
             f'Disabled search for {num_updated} samples in the following {len(updated_families)} families: {family_summary}'
         )
+        try:
+            _trigger_delete_families_dags(from_project, updated_family_dataset_types)
+        except Exception as e:
+            logger_call = logger.warning if isinstance(e, DagRunningException) else logger.error
+            logger_call(str(e))
+
+def _trigger_delete_families_dags(from_project, updated_family_dataset_types):
+    updated_families_by_dataset_type = defaultdict(list)
+    for dataset_type, family_guid in updated_family_dataset_types:
+        updated_families_by_dataset_type[dataset_type].append(family_guid)
+
+    for dataset_type, family_guids in sorted(updated_families_by_dataset_type.items()):
+        trigger_airflow_delete_families(dataset_type, family_guids, from_project)
+        logger.info(f'Successfully triggered DELETE_FAMILIES DAG for {len(family_guids)} family from {from_project.name}/{dataset_type}')
```

Comment:
this command can only be run on a single project so including the project name in the log is a bit redundant/ makes it harder to read. It would be neater as f'Successfully triggered DELETE_FAMILIES DAG for {len(family_guids)} {dataset_type} families'